front/lib/api/assistant/conversation.ts

```diff
@@ -1768,22 +1769,52 @@ export async function postNewContentFragment(
     return cfBlobRes;
   }
 
+  const supersededContentFragmentId = cf.supersededContentFragmentId;
+  // If the request is superseding an existing content fragment, we need to validate
```

Comment:
@flvndvd @Fraggle would love your attention specifically on this part, as it's a bit strange to have the validation here (but at the same time, it would also be weird to pass the conversation to `ContentFragmentResource` and have the validation there.

Reply:
Do we assume that all content fragments can be superseeded? Can't we limit this to web page only (or whatever the type)?

Reply:
Good question.

In my view theres no strong reason to block from here, we could decide to do this for excel files later or any other type of CF.
It's unlikely that we would supersede by mistake right?